src/modules/ShareActionHandlerModule.ts

```diff
@@ -0,0 +1,17 @@
+import {NativeModules} from 'react-native';
+
+const {ShareActionHandlerModule} = NativeModules;
+
+type ShareActionContent = {
+    id: string;
+    content: string;
+    mimeType: string;
+    processedAt: string;
+    aspectRatio: number;
+};
+
+type ShareActionHandlerType = {
```

Comment:
We shouldn't add `Type`, `Interface` suffixes according to TS guidelines

```suggestion
type ShareActionHandler = {
```